fixtures/american-dj/mod-hex100.json

```diff
@@ -3,9 +3,9 @@
   "name": "MOD HEX100",
   "categories": ["Color Changer"],
   "meta": {
-    "authors": ["K", "CGBassPlayer"],
+    "authors": ["K", "Ryan Goodwin"],
     "createDate": "2023-04-24",
-    "lastModifyDate": "2023-04-24"
+    "lastModifyDate": "2023-05-25"
```

Comment:
Please only update the `lastModifyDate` if there are "real" fixture data changes. Updating a link or author doesn't really add value; but e.g. fixing a capability would. So again, please revert the `lastModifyDate` changes in this PR.

And again, would you like to see this documented somewhere?

Reply:
Wasn't sure how strict the `lastModifyDate` was and decided to play it safe. I think a note in the documentation about updating non-fixture related data would be good. I can throw that in and update the PR